code/modules/mob/living/carbon/human/whisper.dm

```diff
@@ -109,24 +109,18 @@
 			message = replacetext(message, "u", "�")
 			message = replacetext(message, "b", "�")
 
-	var/list/listening = hearers(message_range, src)
+	var/list/listening = get_mobs_or_objects_in_view(message_range, src)

```

Comment:
That's how the "broken" version of say works which for some reason Polaris is still using.  And yes the performance impact is drastic.  Recursing through every mob and every object looking for other objects and mobs to recurse in is much slower then just checking turfs and the player list.

https://github.com/Baystation12/Baystation12/blob/dev/code/modules/mob/living/say.dm#L245-L265